src/praisonai/praisonai/cli/legacy/interactive_legacy.py

```
@@ -240,18 +276,22 @@ def display_loop():
                             word_count = len(response.split()) if response else 0
                             q_display = question[:80] + "..." if len(question) > 80 else question
                             
-                            console.print(f"[bold cyan]─── Task #{task_id} completed ({elapsed:.1f}s, {word_count} words) ───[/bold cyan]")
+                            if not streamed:
+                                console.print()  # New line before response
+                            console.print(f"\n[bold cyan]─── Task #{task_id} completed ({elapsed:.1f}s, {word_count} words) ───[/bold cyan]")
                             console.print(f"[bold green]Q:[/bold green] {q_display}")
-                            console.print(f"[bold blue]A:[/bold blue] ", end="")
+                            if not streamed:
+                                console.print(f"[bold blue]A:[/bold blue] ", end="")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)